TWLight/users/oauth.py

```diff
@@ -272,6 +272,12 @@ def authenticate(self, request=None, access_token=None, handshaker=None):
 
         request.session["user_created"] = created
 
+        # Checking if more than one session exists
+        sessions = Session.objects.filter(account_id=user.pk)
+        if sessions.count() >= 1:
```

Comment:
question: could this be written as?
```suggestion
        if sessions.exists():
```

Reply:
That's true! This was a leftover from when I thought I needed to deleted all sessions but the latest.